example/ndpi_util.c

```diff
@@ -650,10 +650,12 @@ void ndpi_workflow_process_packet (struct ndpi_workflow * workflow,
     break;
 
     /* Linux Cooked Capture - 113 */
+#ifdef __linux__
   case DLT_LINUX_SLL :
```

Comment:
I think @numeronove fix is correct. OpenBSD does not support DLT_LINUX_SLL

http://openbsd-archive.7691.n7.nabble.com/redhat-lt-gt-openbsd-tcpdump-td272594.html